src/components/data_comps/datm/cime_config/config_component.xml

```diff
@@ -52,6 +53,7 @@
       <value compset="%CRU">CLMCRUNCEP</value>
       <value compset="%CRUv7">CLMCRUNCEPv7</value>
       <value compset="%GSWP3v1">CLMGSWP3v1</value>
+      <value compset="%NLDAS">CLMNLDAS</value>
```

Comment:
Have you checked that this still works with NLDAS2?  I'm a little concerned about how compset matching works.

Reply:
@jonbob can you comment here?

Reply:
I did not test that, but can

Reply:
except we don't have any support for NLDAS2 in E3SM. You may have to test it in CESM with a merge of this branch

Reply:
You should be able to test this by using a compset long name, or by introducing a temporary compset for the sake of testing.

I'm hopeful that this particular instance will be okay, because of the `match="last"` above, but I'm concerned that there may be other, similar instances. This seems to be begging for problems, if not now, then in the future.

Reply:
Thanks @billsacks - I'll look at that today

Reply:
@billsacks - I had tried this last week but there are files for the NLDAS2 configuration that we do not have

Reply:
They should be available from our public inputdata repository via the check_input_data script. Did you have trouble even when using that?